### PR TITLE
Refactor: Prioritize identify user customer hook

### PR DESCRIPTION
**Pull request by keyur9**
Refactor: Prioritize identify user customer hook
Bump version to 2.0.2

### CODE REVIEW

File: moesifdjango/logger_helper.py [hunk 1]
```diff
@@ -156,69 +156,69 @@ def parse_authorization_header(self, token, field, debug):
...
-                            # Check if payload is not None
-                            if len(split_token) > 1 and split_token[1]:
-                                # Parse and set user Id
-                                user_id = self.parse_authorization_header(split_token[1], field, debug)
-                            else:
-                                # Parse and set user Id
-                                user_id = self.parse_authorization_header(token, field, debug)
+                            # Parse and set user Id
+                            user_id = self.parse_authorization_header(token, field, debug)
+            if not user_id:
```
Comment (dkm199) [inline moesifdjango/logger_helper.py:219]: i think 1) their setting, 2) request.user 3) auth header extraction. @dgilling lemme know if u disagree

Comment (dgilling) [reply moesifdjango/logger_helper.py:219]: Agree, that pri makes most sense.

Comment (dkm199) [reply moesifdjango/logger_helper.py:219]: @keyur9  please fix accordingly

Comment (keyur9) [reply moesifdjango/logger_helper.py]: Have changed the priority and updated the PR

**Review by dkm199 — APPROVED** [review]

**Pull request merged**